src/content/docs/mobile-monitoring/new-relic-mobile-android/troubleshoot/firebase-incompatibility-android.mdx

```diff
@@ -0,0 +1,21 @@
+---
+title: Firebase Incompatibility (Android)
+type: troubleshooting
+tags:
+  - Mobile monitoring
+  - New Relic Mobile Android
+  - Troubleshoot
+metaDescription: 'New Relic Android Agent cannot coexist with firebase PerfMon SDK'
+redirects:
+  - /docs/mobile-monitoring/new-relic-mobile/troubleshoot/firebase-incompatibility-android
+---
+
+## Problem
+
+Conflicts, crashes and errors will show when include both New Relic plugin and Firebase PerfMon SDK within an app.
+
+## Solution
+
+New Relic Android Agent and FireBase PerfMon cannot coexist and be enabled at the same time, it will create an infinite loop in the instrumented code. We would recommend to disable either one to avoid this issue.
```

Comment:
This is not completely true: for the most part Firebase and our agent co-exist just fine. Keep in mind that FB is made up of many packages and I've found it is on the the Performance SDK we have trouble with, and with that only certain features that involve class rewriting. The goal of the story was to isolate and call-out what we can't work with. However, it's probably better at the moment to simply say, "_use one or the other, not both_".